third_party/move/move-compiler-v2/tests/testsuite.rs

```diff
@@ -717,6 +717,20 @@ const TEST_CONFIGS: Lazy<BTreeMap<&str, TestConfig>> = Lazy::new(|| {
             dump_bytecode: DumpLevel::EndStage,
             dump_bytecode_filter: Some(vec![FILE_FORMAT_STAGE]),
         },
+        // Unreachable code remover
```

Comment:
This comment does not make sense. Maybe it is a copy-paste error?

Reply:
done